tests/local-values.yml

```diff
@@ -0,0 +1,75 @@
+nodeCount: &nodeCount 1
```

Comment:
Is this for local testing?

Reply:
Yes

Reply:
Just posting here the discussion Taylor and I had offline:
The concern was fundamentally changing an existing cluster when running this make command but it seems like it already previously had this intended affect. It's out of the scope of this PR but we should look into changing up the Makefile to better suit our needs.